assignment/bundles/resources/assignment.properties

```diff
@@ -1078,3 +1077,18 @@ option.hidepoints=Hide point values (feedback only)
 option.studentpreview=Hide Rubric preview from student
 grading_rubric=Grading Rubric
 asn.list.userubric=This assignment could be graded using a Rubric
+
+# table toolbar common messages
+view.label=View
+gen.viewing.phrase=Viewing {0} - {1} of {2} items
+gen.show=Show {0} items...
+# {0} is the pagesize
+gen.previous.pagesize = Previous {0}
+# {0} is the pagesize
+gen.next.pagesize = Next {0}
+gen.select.label = Choose page size
+gen.listnavselect = To operate the combo box, first press Alt+Down Arrow to open it, and then use the up and down arrow keys to scroll through the options.
+search.label=Search
+search.button=Search
+search.clear=Clear Search
```

Comment:
There are likely orphaned messages in this file now that you've replaced the pager messages, which can probably all be removed in all *.properties files in assignments. Ex:

`list.show = Show` is probably no longer used in the project.